2021-10-28 11:20:45,851 INFO    Thread-3  :3351048 [wandb_setup.py:_flush():69] setting env: {'root_dir': '/grid/koo/home/ztang/QuantPred', 'sweep_id': 'zllo6ess', 'entity': 'ambert', 'project': 'center_bpnet_bin_loss', 'run_id': 'yqbdnmky', 'sweep_param_path': '/grid/koo/home/ztang/QuantPred/wandb/sweep-zllo6ess/config-yqbdnmky.yaml'}
2021-10-28 11:20:45,851 INFO    Thread-3  :3351048 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-28 11:20:45,851 INFO    Thread-3  :3351048 [wandb_init.py:_log_setup():337] Logging user logs to /grid/koo/home/ztang/QuantPred/wandb/run-20211028_112045-yqbdnmky/logs/debug.log
2021-10-28 11:20:45,851 INFO    Thread-3  :3351048 [wandb_init.py:_log_setup():338] Logging internal logs to /grid/koo/home/ztang/QuantPred/wandb/run-20211028_112045-yqbdnmky/logs/debug-internal.log
2021-10-28 11:20:45,852 INFO    Thread-3  :3351048 [wandb_init.py:init():370] calling init triggers
2021-10-28 11:20:45,852 INFO    Thread-3  :3351048 [wandb_init.py:init():377] wandb.init called with sweep_config: {'bin_size': 128, 'data_dir': 'datasets/complete/peak_centered/i_3072_w_1/', 'l_rate': 0.001, 'loss_fn': 'pearsonr_mse', 'model_fn': 'bpnet', 'num_epochs': 100, 'window_size': 2048}
config: {}
2021-10-28 11:20:45,852 INFO    Thread-3  :3351048 [wandb_init.py:init():419] starting backend
2021-10-28 11:20:45,852 INFO    Thread-3  :3351048 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-28 11:20:45,866 INFO    Thread-3  :3351048 [backend.py:ensure_launched():135] starting backend process...
2021-10-28 11:20:45,875 INFO    Thread-3  :3351048 [backend.py:ensure_launched():140] started backend process with pid: 3351209
2021-10-28 11:20:45,876 INFO    Thread-3  :3351048 [wandb_init.py:init():424] backend started and connected
2021-10-28 11:20:45,879 INFO    Thread-3  :3351048 [wandb_run.py:_config_callback():872] config_cb None None {'bin_size': 128, 'data_dir': 'datasets/complete/peak_centered/i_3072_w_1/', 'l_rate': 0.001, 'loss_fn': 'pearsonr_mse', 'model_fn': 'bpnet', 'num_epochs': 100, 'window_size': 2048}
2021-10-28 11:20:45,880 INFO    Thread-3  :3351048 [wandb_init.py:init():472] updated telemetry
2021-10-28 11:20:45,880 INFO    Thread-3  :3351048 [wandb_init.py:init():491] communicating current version
2021-10-28 11:20:46,745 INFO    Thread-3  :3351048 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-28 11:20:46,745 INFO    Thread-3  :3351048 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-10-28 11:20:46,854 INFO    Thread-3  :3351048 [wandb_init.py:init():529] starting run threads in backend
2021-10-28 11:20:48,733 INFO    Thread-3  :3351048 [wandb_run.py:_console_start():1623] atexit reg
2021-10-28 11:20:48,735 INFO    Thread-3  :3351048 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.REDIRECT
2021-10-28 11:20:48,737 INFO    Thread-3  :3351048 [wandb_run.py:_redirect():1502] Redirecting console.
2021-10-28 11:20:48,742 ERROR   Thread-3  :3351048 [wandb_run.py:_redirect():1561] Failed to redirect.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/wandb_run.py", line 1554, in _redirect
    out_redir.install()
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.6/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2021-10-28 11:20:48,746 INFO    Thread-3  :3351048 [wandb_init.py:init():554] run started, returning control to user process
2021-10-28 22:22:05,166 INFO    Thread-3  :3351048 [wandb_run.py:finish():1293] finishing run ambert/center_bpnet_bin_loss/yqbdnmky
